src/aio.rs

```diff
@@ -0,0 +1,265 @@
+// Copyright (C) 2019 Alibaba Cloud Computing. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Safe wrapper for Linux AIO API.
+
+#![allow(non_camel_case_types)]
+
+/* automatically generated by rust-bindgen and then manually edited */
```

Comment:
Please specify version/commit from which you generated the file and which header file was used for generating it.